lib/msf/core/db_manager/http/servlet/credential_servlet.rb

```diff
@@ -25,13 +25,18 @@ def self.get_credentials
       begin
         sanitized_params = sanitize_params(params)
         data = get_db.creds(sanitized_params)
-
+        includes = [:logins, :public, :private, :realm]
         # Need to append the human attribute into the private sub-object before converting to json
         # This is normally pulled from a class method from the MetasploitCredential class
+        response = []
+        data.each do |cred|
+          json = cred.as_json(include: includes).merge('private_class' => cred.private.class.to_s)
```

Comment:
Could we use the `key: value` hash syntax we've been using mostly throughout our code here?